features/ios/support/utils.rb

```diff
@@ -0,0 +1,91 @@
+def turn_off_bluetooth
+	# not yet implemented
+end
+
+def give_location_permission
+	# after launching the application there might be pop-up for permission location access
+	if device_agent.springboard_alert_visible?
+		puts "- Needs to give permission, let's give a permission"
+
+		begin
+			device_agent.dismiss_springboard_alert("Always Allow")
+		rescue
+			device_agent.dismiss_springboard_alert("Allow")
+		end
+	end
+end
+
+def allow_notification
+	# after launching the application there might be pop-up for permission
+	if device_agent.springboard_alert_visible?
+		puts "- Needs to give permission, let's give a permission"
+
+		begin
+			device_agent.dismiss_springboard_alert("Allow")
+		rescue
+			device_agent.dismiss_springboard_alert("Always Allow")
+		end
+	end
+end
+
+def allow_location
```

Comment:
Wrap these methods in a module to avoid defining them in the Kernel. 

```
require "cucumber"

module Calabash
  module Xplatform
   # Add methods here
  end
end

World(Calabash::Xplatform)
```